aap-common/proc-aap-pull-command-container-image.adoc

```diff
@@ -6,7 +6,7 @@ Pull the Docker image for the Ansible on Clouds operational container with the s
 
 [NOTE]
 ====
-Before Pulling the docker image, make sure you are logged in to redhat registry using docker. Use the following command to login to registry.redhat.com.
+Before Pulling the docker image, make sure you are logged in to Red Hat registry using docker. Use the following command to login to registry.redhat.com.
```

Comment:
Perhaps it would be better not to use two names for the same thing. Red Hat registry =registry.redhat.com. here and in subsequent entries